src/dlls/mscordac/CMakeLists.txt

```diff
@@ -184,4 +184,21 @@ endif(WIN32)
 target_link_libraries(mscordaccore PRIVATE ${COREDAC_LIBRARIES})
 
 # add the install targets
-install_clr(mscordaccore)
+install_clr(TARGETS mscordaccore)
+install_clr(TARGETS mscordaccore DESTINATION sharedFramework)
+if(WIN32)
+    set(LONG_NAME_HOST_ARCH ${CLR_CMAKE_HOST_ARCH})
+    if (CLR_CMAKE_PLATFORM_ARCH_AMD64)
+        set(LONG_NAME_HOST_ARCH "amd64")
+    endif()
+
+    set(LONG_NAME_TARGET_ARCH ${CLR_CMAKE_TARGET_ARCH})
+    if (CLR_CMAKE_PLATFORM_ARCH_AMD64)
```

Comment:
These two `if` clauses are using the same invariant, I would put them in the same branch.